app/code/Magento/Quote/Test/Unit/Model/Quote/Address/Total/SubtotalTest.php

```diff
@@ -110,10 +110,14 @@ public function testCollect($price, $originalPrice, $itemHasParent, $expectedPri
             ]
         );
         /** @var Address|MockObject $address */
-        $address = $this->createPartialMock(
-            Address::class,
-            ['setTotalQty', 'getTotalQty', 'removeItem', 'getQuote']
-        );
+        $address = $this->getMockBuilder(Address::class)
+            ->disableOriginalConstructor()
+            ->disableOriginalClone()
+            ->disableArgumentCloning()
```

Comment:
**YAGNI** If original code is not performing any `clone` then you don't need to provide these methods.